org.freedesktop.Sdk.Extension.dotnet7.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: org.freedesktop.Sdk.Extension.dotnet7
+branch: '22.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+modules:
+  - name: dotnet-sdk
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /usr/lib/sdk/dotnet7/{bin,lib}
+      - cp -r * /usr/lib/sdk/dotnet7/lib
+      - ln -s /usr/lib/sdk/dotnet7/lib/dotnet /usr/lib/sdk/dotnet7/bin/dotnet
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        url: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/7.0.100/dotnet-sdk-7.0.100-linux-x64.tar.gz
+        sha256: 5d4a58eeca1a4ab095b55e6486b1278147b0ab121ce883b2bcde3d3626c9a81c
+        x-checker-data:
+          type: html
+          url: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/7.0/latest.version
+          version-pattern: ^([\d\.a-z-]+)$
+          url-template: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$version/dotnet-sdk-$version-linux-x64.tar.gz
+      - type: archive
+        only-arches: [aarch64]
+        url: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/7.0.100/dotnet-sdk-7.0.100-linux-arm64.tar.gz
+        sha256: ef675d0b2f3012b8ac0c67952c752f8646decf3d68713b17d7bf45dfaf476441
+        x-checker-data:
+          type: html
+          url: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/7.0/latest.version
+          version-pattern: ^([\d\.a-z-]+)$
+          url-template: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$version/dotnet-sdk-$version-linux-arm64.tar.gz
+  - name: scripts
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /usr/lib/sdk/dotnet7/share/appdata
+      - cp install.sh install-sdk.sh /usr/lib/sdk/dotnet7/bin
+      - cp enable.sh /usr/lib/sdk/dotnet7
+      - cp org.freedesktop.Sdk.Extension.dotnet7.appdata.xml ${FLATPAK_DEST}/share/appdata
+      - appstream-compose --basename=org.freedesktop.Sdk.Extension.dotnet7 --prefix=${FLATPAK_DEST}
```

Comment:
Why did you need this? flatpak-builder should do this automatic.

Reply:
Because it doesn't for extensions.

Reply:
Maybe it is better to use appstreamcli here instead of the old appstream-compose

Reply:
we use the same as elsewhere. Every other extension do that.